specification/README.md

```diff
@@ -57,14 +57,14 @@ The suggested architecture is composed of the following sub-projects:
         - Express Controllers/Routes layer - this component houses all the express routing logic and exposes a valid REST CRUD API for our front-end to consume.
         - Service Layer - this is the home of all our data storage and validation logic. If we want to store something in our database, we should go through here.
 
-- Data storage - this is the persistent storage for our system - in our case this is going to be the Mongo Database.
+- Data storage - this is the persistent storage for our system.
 
 ## General pointers
 
-These ideas bellow are recommendations, that the authors think may be of use to the developers of this system.
+Authors think bellow recommendations may be of use to the developers of this system.
```

Comment:
Maybe the would be better - `In the authors opinion, the`?

Reply:
Addressed [34f7848](https://github.com/Teodor92/Project-Company-Life/commit/34f78483e4d90ec80582963bade220c817b88b57)

Reply:
Addressed [3a4dc4f](https://github.com/Teodor92/Project-Company-Life/commit/3a4dc4f4322e0127a07ddaa7486209e95e4c0b85)